src/main/java/org/renci/exogeni/image/service/oscontroller/OsImageController.java

```diff
@@ -176,7 +182,65 @@ private String auth(String region) throws Exception {
      *
      * @throws exception in case of error
      */
-    public ImageDetails getImage(String imageId) throws Exception {
+    public Integer getImageDetails(String imageId) throws Exception {
+
+        if (region == null || imageId == null) {
+            throw new ImageServiceException("Failed to construct image request; invalid input params");
+        }
+        try {
+            String token = auth(region);
+            if (token != null) {
+                HttpHeaders headers = new HttpHeaders();
+                System.out.println("Setting token " + token);
```

Comment:
Maybe get rid or replace with logger call?

Reply:
ok, will change to logger statement, I didn't have to use the command line tool, instead i could pass Range in the Header to download parts of the file. I have configured the python client to by default to download in parts of 50MB increments. If RANGE is not specified in the header the whole file is downloaded at once.